### PR TITLE
Adding Component and Integration Tests

### PR DESCRIPTION
**Pull request by AMCR**
#### Label: ####
_improvement_
#### What's this PR do?
Adding Component and Integration Tests
#### Where should the reviewer start?
Component-Tests
#### What tests were added?
CT/IT
#### Any background context you want to provide?
UCM Content Enrichment
#### Outstanding questions?
None
#### Definition of Done:
- [ ] Is there appropriate test coverage?
- [ ] Does the confluence need to be updated?
- [ ] Does this add new dependencies? If so, have the healthchecks been updated?
- [ ] Any Deployment changes we need to be aware of? New infrastructure? Changes to existing components?
- [ ] Is there appropriate logging included?
- [ ] Does it imply any kind of data migration? If so what are the steps for that?
- [ ] Is the Documentation been updated, if needed?

### CODE REVIEW

File: component-test/src/main/scala/uk/co/telegraph/ucm/enricher/utils/package.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+package object utils {
+
+  def fromPayload(name:String):String = {
+    Source.fromInputStream( getClass.getResourceAsStream(name) )(Codec.UTF8)
+      .getLines()
+      .mkString("", "\n", "\n")
+  }
+
+//  def queryDockerPort(command:String):Option[Int] = {
```
Comment (athermughal) [inline component-test/src/main/scala/uk/co/telegraph/ucm/enricher/utils/package.scala]: Is that needed any more?

File: component-test/src/test/scala/uk/co/telegraph/ucm/enricher/message/MessageServiceSpecs.scala [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+      messagingComponent.triggerEnrichWithoutNotification(contentId)
+      eventually{
+        storageComponent.checkExistEnrichItemById     (contentId)
+        messagingComponent.hasMessagesWithInDlq       (contentId) shouldBe false
+        notificationComponent.hasPublishedMessagesWith(contentId) shouldBe false
+      }
+    }
+
+//    scenario("should get an error message if the element does not exist", ctAndIt:_*) {
```
Comment (athermughal) [inline component-test/src/test/scala/uk/co/telegraph/ucm/enricher/message/MessageServiceSpecs.scala]: Is that needed any more?

Comment (AMCR) [reply component-test/src/test/scala/uk/co/telegraph/ucm/enricher/message/MessageServiceSpecs.scala]: The test was not ready.

File: component-test/src/main/scala/uk/co/telegraph/ucm/enricher/utils/package.scala (hunk 1 above)
Comment (AMCR) [reply component-test/src/main/scala/uk/co/telegraph/ucm/enricher/utils/package.scala]: Removed

**Pull request merged**